testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -893,3 +893,11 @@ def check_operator(op, method, level):
         for op, method in operators:
             for level in levels:
                 yield check_operator, op, method, level
+
+
+class TestAtomGroup(object):
```

Comment:
Sure? I usually like that filename -> test_filename convention to find tests for functions.